docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,323 @@
+| "jsonrpc" | A string specifying the version of JSON RPC protocol being used. Must be exactly **"2.0"** currently in all versions of SDL Core.|
+| "method" | A String containing the information of the method to be invoked. The format is `[componentName].[methodName]`.|
+| "params" | A structured object that holds the parameter values to be used during the invocation of the method. This property may be omitted.|
+
+### Example Requests
+#### Request with no Parameters
+```json
+{
+  "id": 125,
+  "jsonrpc": "2.0",
+  "method": "Buttons.GetCapabilities"
+}
+```
+#### Request with Parameters
+```json
+{
+  "id": 92,
+  "jsonrpc": "2.0",
+  "method": "UI.Alert",
+  "params": {
+    "alertStrings": [
+      {
+        "fieldName": "alertText1",
+        "fieldText": "WARNING"
+      },
+      {
+        "fieldName": "alertText2",
+        "fieldText": "Adverse Weather Conditions Ahead"
+      }
+    ],
+    "duration": 4000,
+    "softButtons": [
+      {
+        "type": "TEXT",
+        "text": "OK",
+        "softButtonID": 697,
+        "systemAction": "STEAL_FOCUS"
+      }
+    ],
+    "appID": 8218
+  }
+}
+```
+
+## Notification
+A notification is a request object without an `id` property. For all the other properties, see the Request Section above.
+
+The receiver should not reply to a notification, i.e. no response object needs to be returned to the client upon receipt of a notification.
+
+### Example Notifications
+#### Notification with no Parameters
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "UI.OnReady"
+}
+```
+#### Notifications with Parameters
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "BasicCommunication.OnAppActivated",
+  "params": {
+    "appID": 6578
+  }
+}
+
+{
+  "jsonrpc": "2.0",
+  "method": "Buttons.OnButtonPress",
+  "params": {
+    "mode": "SHORT",
+    "name": "OK"
+  }
+}
+```
+
+## Response
+On receipt of a request message, the server must reply with a response. The response is expressed as a single JSON Object with the following properties.
+
+| Property | Description    |
+| :------------- | :------------- |
+| "id"      | Required property which must be the same as the value of the associated request object. If there was an error in detecting the id in the request object, this value must be null  |
+|"jsonrpc"| Must be exactly **"2.0"**|
+|"result"| Required on success. Must not exist if there was an error invoking the method. The result property must contain a `method` field which is the same as the corresponding request, a `code` field with **0** to indicate success or **21** to indicate success with warnings.  No other [result codes](../common/enums/#result) should be sent in the result property. The result property may also include additional properties as defined in the HMI API.|
```

Comment:
```suggestion
|"result"| Required on success. Must not exist if there was an error invoking the method. The result property must contain a `method` field which is the same as the corresponding request and a `code` field with **0** to indicate success or **21** to indicate success with warnings. No other [result codes](../common/enums/#result) should be sent in the result property. The result property may also include additional properties as defined in the HMI API.|
```

Inserted `and` between needing a method and needing a code
Extra space before `No other [resul`

I think I mentioned this before, but the language 'No other result codes should be sent in the result property' just isn't accurate. I know we use it a lot and I believe it used to be true, but there are like 7 codes treated as success these days.

Reply:
Good point ill remove that